plugins/vpp/l3plugin/vppcalls/l3_vppcalls.go

```diff
@@ -232,6 +236,15 @@ type TeibVppRead interface {
 	DumpTeib() ([]*l3.TeibEntry, error)
 }
 
+// VrrpVppAPI provides methods for managing VPP VRRP.
+type VrrpVppAPI interface {
+	VppAddVrrp(entry *l3.VRRPEntry) error
+	VppDelVrrp(entry *l3.VRRPEntry) error
+	VppStartVrrp(entry *l3.VRRPEntry) error
+	VppStopVrrp(entry *l3.VRRPEntry) error
+	DumpVrrpEntries() ([]*l3.VRRPEntry, error)
+}
```

Comment:
@milanlenco Could you tell me whether other [API](https://docs.fd.io/vpp/21.01/d3/dbe/vrrp_8api.html) methods should be implemented (peer and track_if) or these are sufficient for now?